sri/sri.go

```diff
@@ -0,0 +1,83 @@
+// hashes.
+// https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity
+package sri
+
+import (
+	"crypto/sha256"
+	"crypto/sha512"
+	b64 "encoding/base64"
+	"fmt"
+	"hash"
+	"io"
+	"strings"
+	b64 "encoding/base64"
+)
+
+// Algo is a supported hashing algorithm
+type Algo string
+
+const (
+	// SHA256 algo
+	SHA256 = Algo("sha256")
+	// SHA384 algo
+	SHA384 = Algo("sha384")
```

Comment:
I have never seen sha384 in the wild. Any good reason to support it?

Reply:
Mainly because the spec lists all the supported algos and it's in that list. I agree that it's not really common.